hostports/watcher.go

```diff
@@ -100,13 +100,16 @@ func (p PortRule) natIptables() []byte {
 	if p.SourceIP == "0.0.0.0" {
 		buf.WriteString(fmt.Sprintf("\n-A CATTLE_PREROUTING -p %v -m %v --dport %v -m addrtype --dst-type LOCAL -j DNAT --to-destination %v:%v",
 			p.Protocol, p.Protocol, p.SourcePort, p.TargetIP, p.TargetPort))
+
+		buf.WriteString(fmt.Sprintf("\n-A CATTLE_OUTPUT     -p %v -m %v --dport %v -m addrtype --dst-type LOCAL -j DNAT --to-destination %v:%v",
```

Comment:
Can you please get rid of the extra spaces `CATTLE_OUTPUT     -p`?

Reply:
No problem, the extra space is just for aligning the parameters with the same command on CATTLE_PREROUTING.